pkg/streamingpromql/types/operator.go

```diff
@@ -63,14 +63,7 @@ type RangeVectorOperator interface {
 	// NextStepSamples populates the provided RingBuffers with the samples for the next time step for the
```

Comment:
NextStepSamples populates the internal RingBuffers with the samples for the next time step for the current series and returns the values in RangeVectorStepData, or returns EOS if no more time

Reply:
Good catch, fixed in 489fe40